monitor-openqa_job

```diff
@@ -18,38 +11,72 @@ obs_component="${obs_component:-"package"}"
 obs_package_name="${1:-""}"
 comment_on_obs=${comment_on_obs:-}
 
-# shellcheck source=/dev/null
-. "$(dirname "$0")"/_common
-
-[[ -f job_post_response ]] || (echo "Need job response status file 'job_post_response'" && exit 2)
-declare -A failed_versions
-failed_jobs=()
-for job_id in $(job_ids job_post_response); do
-    echo "Waiting for job $job_id to finish"
-    while sleep "${sleep_time}"; do
-        job_state=$($openqa_cli api --host "$host" jobs/"$job_id" | jq -r '.job.state')
-        [[ $job_state = 'done' ]] && break
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
```

Comment:
Maybe add an example here on how to invoke this script (how to supply job id(s?)).